src/GeneralStateTestsFiller/stEWASMTests/createFailFiller.yml

```diff
@@ -0,0 +1,138 @@
+
+          (data (i32.const 224) "\a0\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\01")
+
+          (export "memory" (memory 0))
+          (export "main" (func $main))
+
+          (func $main
+            ;; locals
+            (local $codeMemLoc i32)
+            (local $codeLen i32)
+            (local $addrOffset i32)
+            (local $memPointerStorageKey1 i32)
+            (local $memPointerStorageKey2 i32)
+            (local $memPointerCreateValue i32)
+            (local $memPointerCreateResult i32)
+
+            ;; Location and length of code stored in (data) section above
+            (set_local $codeMemLoc (i32.const 256))
+            (set_local $codeLen (i32.const 91))
+            (set_local $addrOffset (i32.const 224))
+              
+            ;; memory layout and pointers
+            (set_local $memPointerStorageKey1 (i32.const 0))
+            (set_local $memPointerStorageKey2 (i32.const 32))
+            (set_local $memPointerCreateValue (i32.const 64))
+            (set_local $memPointerCreateResult (i32.const 96))
+
+            ;; get external code
+            (set_local $codeLen 
+              (call $getExternalCodeSize (get_local $addrOffset)))
+
+            (call $externalCodeCopy
+              (get_local $addrOffset)  ;; address
+              (get_local $codeMemLoc)  ;; result
+              (i32.const 0)            ;; code offset
+              (get_local $codeLen))    ;; code length
+            
+            ;; initialize
+            ;; call create with value transfer 0x1234
+            (i32.store (get_local $memPointerCreateValue) (i32.const 0x1234))
+
+            ;; call CREATE and store the return success/failure value
+            (call $storageStore
+              (get_local $memPointerStorageKey1)
+              (call $create
+                ;; value offset
+                (get_local $memPointerCreateValue)
+                ;; data offset
+                (get_local $codeMemLoc)
+                ;; data length
+                (get_local $codeLen)
+                ;; result offset (new contract address)
+                (get_local $memPointerCreateResult)
+              )
+            )
+
+            ;; store the result (new contract address)
+            (call $storageStore
+              (get_local $memPointerStorageKey2)
+              (get_local $memPointerCreateResult)
+            )
+          )
+        )
+  expect:
+    - indexes:
+        data: !!int -1
+        gas: !!int -1
+        value: !!int -1
+      network:
+        - ALL
+      result:
+        a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          balance: '99993098524'
+        deadbeef00000000000000000000000000000000:
+          storage: {
+            # Expect CREATE to return success
+            # $memPointerStorageKey1 = 0
+            0: '0x0000000000000000000000000000000000000000000000000000000000000000',
```

Comment:
Does this mean it returns success? That seems to be a bug somewhere (possibly hera).